src/api/app/views/webui/package/bottom_actions/_request_devel_project_change.html.haml

```diff
@@ -1,4 +1,4 @@
 %li.list-inline-item
-  = link_to('#', class: 'nav-link', data: { toggle: 'modal', target: '#change-devel-request-modal' }) do
-    %i.fas.fa-exchange-alt.text-primary
-    Request devel project change
+  = link_to(new_project_package_devel_project_change_path(package.project, package), class: 'nav-link') do
+    %i.fas.fa-exchange-alt.fa-lg.mr-2
```

Comment:
This icon is now larger than other bottom actions. The margin on the right is also not needed to be consistent with other bottom actions. I agree with the removal of `.text-primary`. This is a secondary action.

![larger-icon](https://user-images.githubusercontent.com/1102934/83143891-eb64c100-a0f2-11ea-91f8-a7ba17f3f068.png)

```suggestion
    %i.fas.fa-exchange-alt
```

Reply:
Applied, thanks.